include/grpc/credentials.h

```diff
@@ -485,11 +485,21 @@ GRPCAPI grpc_call_credentials* grpc_composite_call_credentials_create(
     grpc_call_credentials* creds1, grpc_call_credentials* creds2,
     void* reserved);
 
+/** Context that can be used by the google compute engine create credentials api
+   in order to configure the desired credentials. */
+typedef struct {
+  /** Indicates if the credentials should be used on alts connections. */
+  bool create_alts_credentials;
```

Comment:
What do you think if we accept directly query params here? So the C-Core API would be agnostic to ALTS. This is fine since only C++ uses it. Wrapper languages create grpc_call_credentials in their own stack.